### PR TITLE
Create evaluation profile for console and console backend service

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Create evaluation profile for console and console backend service

**Related issue(s)**

### CODE REVIEW

**Comment by Sawthis** [comment]
It resolves the #9898 issue.

**Review by parostatkiem — COMMENTED** [review]
What about the `cpu` limits/requests?

File: resources/console/charts/web/templates/deployment.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@ spec:
         - name: console
           image: "{{ .Values.global.containerRegistry.path }}/console:{{ .Values.console.image.tag }}"
           imagePullPolicy: {{ .Values.console.image.pullPolicy }}
+          resources: 
+            {{- toYaml .Values.deployment.resources | nindent 12 }}
```
Comment (parostatkiem) [inline resources/console/charts/web/templates/deployment.yaml:39]: what about the `core-ui` container? And other "frontend" deployments like `log-ui` or `service-catalog-io`?

File: resources/console/profile-evaluation.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+web:
+  deployment:
+    resources:
+      requests:
+        memory: 16Mi
```
Comment (parostatkiem) [inline resources/console/profile-evaluation.yaml:5]: On two different clusters I checked, the pod from this deployment consumes exactly 19Mi (2Mi Console; 17Mi Core-ui) while running. I think there's no case when it'd take less, so I suggest to use this value as the `request`.

File: resources/console/profile-evaluation.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+      limits:
+        memory: 32Mi
+backend: 
+  deployment:
+    resources:
+      requests:
+        memory: 32Mi
```
Comment (parostatkiem) [inline resources/console/profile-evaluation.yaml:12]: On two different clusters I checked, the pod from this deployment consumes 104Mi and 116Mi at a `standby` state. I think we should request a value closer to these numbers.

File: resources/console/profile-production.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    resources:
+      requests:
+        memory: 64Mi
+      limits:
+        memory: 128Mi
+backend: 
+  deployment:
+    resources:
+      requests:
+        memory: 64Mi
```
Comment (parostatkiem) [inline resources/console/profile-production.yaml:12]: ^ As in the profile-evaluation case - I think it'll never consume less than ~100Mi.

File: resources/console/profile-evaluation.yaml (hunk 3 above)
Comment (parostatkiem) [reply resources/console/profile-evaluation.yaml:12]: Here's the 12-hours `weekly` cluster summary from Grafana:
image

File: resources/console/profile-evaluation.yaml (hunk 2 above)
Comment (parostatkiem) [reply resources/console/profile-evaluation.yaml:5]: Also, here's the `weekly` cluster 12h summary. A bit different figures but still, I think it should be much less for the `console` container (as you currently apply) or much more for the whole pod.
image

**Review by parostatkiem — CHANGES_REQUESTED** [review]

File: resources/console/profile-evaluation.yaml (hunk 2 above)
Comment (dariadomagala) [reply resources/console/profile-evaluation.yaml:5]: This is the evaluation profile, so weekly is not a good source of information. This cluster runs lots of tests all the time, and we won't have those tests in the evaluation profile.

File: resources/logging/profile-evaluation.yaml [hunk 5]
```diff
@@ -17,3 +17,12 @@ fluent-bit:
...
       cpu: 20m
       memory: 40Mi
+
+logui:
+  resources: 
+    requests:
+      cpu: 1m
+      memory: 4Mi
+    limits:
+      cpu: 1m
```
Comment (parostatkiem) [inline resources/logging/profile-evaluation.yaml:27]: Look what few page reloads have done. I think `3m` value would be better.
image

**Review by parostatkiem — CHANGES_REQUESTED** [review]

**Review by parostatkiem — APPROVED** [review]
🌵

File: resources/console/profile-evaluation.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      limits:
+        cpu: 10m
+        memory: 32Mi
+backend:
+  resources:
+    requests:
+      cpu: 2m
+      memory: 40Mi
+    limits:
+      cpu: 10m
```
Comment (adamwalach) [inline resources/console/profile-evaluation.yaml:24]: I think limit values for CPU are a bit to low, take a look on values of other components (search for "Describe nodes (after installation)"):
https://storage.googleapis.com/kyma-prow-logs/logs/adamwalach_test_of_prowjob_kyma-integration-evaluation-gardener/1334523088292810752/build-log.txt
Maybe we could set it to a value closer to `50m`?
That may improve startup time as well
That comment is about all cpu limits -  for other components as well.

**Comment by dariadomagala** [comment]
PR recreated here - https://github.com/kyma-project/kyma/pull/10133

**Pull request closed without merging**